wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -28,6 +28,9 @@ <h2>{{ postelection.friendly_name }}</h2>
         {% else %}
           {{ postelection.election.in_past|yesno:"was,will be" }} held
           <strong>{{ postelection.election.election_date|naturalday:"\o\n l j F Y" }}</strong>.
+          {% if postelection.next_ballot %}
```

Comment:
Being in the `_single_ballot` partial, this will be called for all current elections too. I don't think that's helpful, as we know that a current ballot wont have a newer election.

Can you move this call to the `post_view.html` template directly, or if we want it to show in the card, add a conditional flag if we want to make this call or not?

In general, I try to be careful about adding any extra queries to the pages that get the most traffic - every little helps!

Reply:
Thought it best to do both - remove from the partial, but also add in the `current` check in case it does get moved there or elsewhere

Reply:
Yeah, good idea :+1: